google-cloud-util/google-cloud-compat-checker/pom.xml

```diff
@@ -21,6 +21,7 @@
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-tcnative-boringssl-static</artifactId>
+      <version>2.0.7.Final</version>
```

Comment:
@garrettjonesgoogle If netty is already shaded, do we need the compat checker at all?

Reply:
There are fewer issues for it to check, but it will still tell you if your platform isn't supported.